src/SwipeableListItem.js

```diff
@@ -250,21 +256,24 @@ class SwipeableListItem extends PureComponent {
           }
           break;
       }
+
+      if (this.props.onSwipeStart && this.isSwiping()) {
+        this.props.onSwipeStart();
+      }
     }
   };
 
-  isSwiping = () =>
-    this.dragStartedWithinItem() &&
-    (this.dragDirection === DragDirection.LEFT ||
-      this.dragDirection === DragDirection.RIGHT);
-
-  updatePosition = () => {
+  isSwiping = () => {
     const { blockSwipe } = this.props;
+    return (
```

Comment:
Consider spacing the return to aid readability.

```suggestion

    return (
```